CHANGELOG.md

```diff
@@ -1,5 +1,13 @@
 # Changelog
 
+## 0.8.0
+
+* Fix HTMPredictionModel prediction using SDRClassifier (#3665)
+* Allow multiple classifications for each record to SDRClassifier (#3669)
+* Remove CLAClassifier (#3665)
+* Add capnp serialization to TMRegion (#3657)
+* Document ExperimentDescriptionAPI (#3679)
+
```

Comment:
I don't really understand but I can see f83fb8954 in my commit history so the change is definitely there, just not sure where 96dcdac is.